### PR TITLE
Feature/create articles per user

### PR DESCRIPTION
**Pull request by mixamum**

### CODE REVIEW

File: packages/server/src/server.ts
Comment (prasenjeet-symon) [inline packages/server/src/server.ts:1]: There is no need to touch this file. Please all the coded added by you from here .

File: packages/server/src/seeder/utils.ts [hunk 1]
```diff
@@ -147,6 +162,26 @@ export async function createArticlesPerUser(email: string, numberOfArticles: num
     // Create articles to the database
     // use the function generateFakeArticle to generate fake article
     // initially article should be created in draft form ( isPublished status set to false )
+
+    for (let i = 0; i < numberOfArticles; i++) {
+        const fakeArticle = await generateFakeArticle();
+        axios.post('/api/user/article', {
```
Comment (prasenjeet-symon) [inline packages/server/src/seeder/utils.ts:168]: Do not call the axios , use the Prisma client directly to add the data to database.

File: packages/server/src/seeder/utils.ts [hunk 2]
```diff
@@ -3,6 +3,11 @@ import { UniqueEnforcer } from 'enforce-unique';
 import * as readFileFs from 'fs';
 import showdown from 'showdown';
 import { PrismaClientSingleton } from '../utils';
+import { response } from 'express';
+import path from "path"
+import router from '../api'; 
+import axios from 'axios';
```
Comment (prasenjeet-symon) [inline packages/server/src/seeder/utils.ts:9]: del import

File: packages/server/src/seeder/utils.ts [hunk 3]
```diff
@@ -3,6 +3,11 @@ import { UniqueEnforcer } from 'enforce-unique';
 import * as readFileFs from 'fs';
 import showdown from 'showdown';
 import { PrismaClientSingleton } from '../utils';
+import { response } from 'express';
```
Comment (prasenjeet-symon) [inline packages/server/src/seeder/utils.ts:6]: del import

File: packages/server/src/seeder/utils.ts [hunk 4]
```diff
@@ -3,6 +3,11 @@ import { UniqueEnforcer } from 'enforce-unique';
...
+import path from "path"
+import router from '../api';
```
Comment (prasenjeet-symon) [inline packages/server/src/seeder/utils.ts:8]: del import

**Review by prasenjeet-symon — CHANGES_REQUESTED** [review]
You do not have to call the AXIOS here , use the prisma client to add data to database . Thanks

File: packages/server/src/server.ts
Comment (mixamum) [reply packages/server/src/server.ts:1]: Yes I know this, I was testing it. I will remove it on the final

File: packages/server/src/seeder/utils.ts (hunk 1 above)
Comment (mixamum) [reply packages/server/src/seeder/utils.ts:168]: Ok, that was my question, Just wanted to make sure that I use prisma directly

Comment (prasenjeet-symon) [reply packages/server/src/seeder/utils.ts:168]: Yes use prisma directly

File: packages/server/src/seeder/utils.ts [hunk 5]
```diff
@@ -163,23 +160,43 @@ export async function createArticlesPerUser(email: string, numberOfArticles: num
...
-            console.log("ERROR")
-            console.log(error);
-        });
+
+        const title = fakeArticle!.title;
+        const subTitle = fakeArticle!.subtitle;
+        const htmlContent = fakeArticle!.htmlContent;
+        const markdownContent = fakeArticle!.markdownContent;
+
+        await prisma.user.update({
```
Comment (prasenjeet-symon) [inline packages/server/src/seeder/utils.ts:184]: It is fine but if you could use Promise.all it would speed up the things.

**Review by prasenjeet-symon — APPROVED** [review]
Looks good and safe . Thanks for your valuable contribution 🙏

File: packages/server/src/seeder/utils.ts (hunk 5 above)
Comment (mixamum) [reply packages/server/src/seeder/utils.ts:184]: promise.all when getting the values from fakeArticle?

**Pull request merged**